code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -839,12 +839,19 @@
 	result_amount = 2
 
 /datum/chemical_reaction/space_cleaner
-	name = "Space cleaner"
+	name = "Space Cleaner"
 	id = CLEANER
 	result = CLEANER
 	required_reagents = list(AMMONIA = 1, WATER = 1)
 	result_amount = 2
 
+/datum/chemical_reaction/space_cleaner/bleach
```

Comment:
You're not inheriting anything for the purposes of the reaction itself is what I mean, you've overridden all variables